base/system/chkdsk/chkdsk.rc

```diff
@@ -16,9 +16,12 @@
 #ifdef LANGUAGE_FR_FR
     #include "lang/fr-FR.rc"
 #endif
+#ifdef LANGUAGE_RU_RU
```

Comment:
Please keep the lexicographical ordering of the languages: why moving russian upwards?